core/src/commitment.rs

```diff
@@ -123,6 +123,22 @@ impl BlockCommitmentCache {
         self.root
     }
 
+    fn highest_slot_with_confirmation(&self, confirmation_count: usize) -> Slot {
+        assert!(confirmation_count > 0 && confirmation_count <= MAX_LOCKOUT_HISTORY);
+        for slot in (self.root()..self.slot()).rev() {
+            if let Some(count) = self.get_confirmation_count(slot) {
+                if count >= 1 {
```

Comment:
```suggestion
                if count >= confirmation_count {
```

Reply:
🤦‍♀️ Thanks. Added this more-flexible method at the last minute; bad copy-pasta